### PR TITLE
improve comments on Hebrew ccc 18 & 19

### PR DESCRIPTION
**Pull request by bdenckla**
improve comments to reflect that:
ccc 18 includes both "flavors" of qamats: qamats [gadol] and qamats qatan
ccc 19 includes both "flavors" of holam: holam [not haser for vav] and holam haser for vav
This code and its comments may have been written before these code points were added to Unicode.
I.e. originally, these comments may have been complete, but they may have become incomplete in the meantime.

### CODE REVIEW

**Comment by dscorbett** [comment]
I have no opinion as to whether this change is needed, but if it is, the same change should apply to the similar comments in src/hb-ot-shape-fallback.cc.

**Comment by bdenckla** [comment]
> the same change should apply to the similar comments in src/hb-ot-shape-fallback.cc.

Woops... thanks for pointing this out. As a result of your feedback, @dscorbett, I did the additional, analogous changes as an additional commit on top of my first and it somehow automatically updated this pull request, which in this case is what I wanted but I was still a little surprised.... a pull request (at least as implemented by GitHub) is for a (possibly slipping/changing) _branch_, not a _commit_? I guess so.

Hope this is all okay... I can see an argument for asking me to "squashingly rebase" to make this a single commit like it should have been in the first place, but I hope this will suffice. Haven't done a rebase in a long time; am a bit rusty.

**Comment by alerque** [comment]
Yes, PR's are a request to merge a branch, and the branch is mutable. You can add, remove, or amend commits in the branch up until it is merged. That's how you typically deal with PR feedback. The maintainer(s) have the option to merge with or without squash. You can also rebase and squash or rewrite anything yourself as long as the PR is open. This is one of several reasons GitHub recommends each PR be opened from a unique branch that is not your fork's default branch.

Thanks for taking the time to contribute by the way.

**Review by alerque — APPROVED** [review]

**Pull request merged**
